app/uk/gov/hmrc/agentclientrelationships/repository/RelationshipCopyRecordRepository.scala

```diff
@@ -43,7 +43,7 @@ case class RelationshipCopyRecord(
   clientIdentifier: String,
   clientIdentifierType: String,
   references: Option[Set[RelationshipReference]] = None,
-  dateTime: DateTime = now(UTC),
+  zonedDateTime: ZonedDateTime = now(UTC),
```

Comment:
Need to keep the field name the same here as it's use in the (de)serialisation of this record to/from the DB.